### ISSUE TITLE
New theme for Vim

### ISSUE THREAD

**Issue by prateektade** [rose-pine/rose-pine-theme#87]
### Link to theme repo

https://github.com/prateektade/rosepine-vim

### What would you like to do with your repo?

Transfer my repo into the org

### Additional information

I made this as a learning exercise by referring to the [Neovim](https://github.com/rose-pine/neovim) color scheme. It has separate files for all three flavors, and I have also added lightline themes. However, all colors are hard-coded and I also didn't include the integrations of all the Neovim plugins listed in the wiki.

**Comment by mvllow** [comment]
Thank you for adding this, feel free to transfer in! We do have a vim repo already listed on our website that we may want to replace with this once transferred.

**Comment by prateektade** [comment]
Happy New Year and thank you! I am actually looking into making it programmatic and tweaking it a little bit to bring it a lot closer to the neovim repo, but that will take some time. I'll update here once those changes are done, and then transfer it once you've taken a look.

**Comment by prateektade** [comment]
Hello there! I just completed updating the colorscheme using this tool called [vim-colortemplate](https://github.com/lifepillar/vim-colortemplate) which has also been used to rework the colorschemes packaged with Vim. Thanks to this tool I was also able to add support for 256 color terminal emulators.

I have added the integration for the [vimwiki](https://github.com/vimwiki/vimwiki) plugin apart from the [lightline](https://github.com/itchyny/lightline.vim). I haven't added the Neovim plugins because I feel it makes more sense to use our Neovim colorscheme in that case.
